src/service/payment.service.ts

```diff
@@ -3,7 +3,7 @@ import { v4 as uuidv4 } from 'uuid';
 import { Session } from "@companieshouse/node-session-handler";
 import ApiClient from "@companieshouse/api-sdk-node/dist/client";
 import { isActiveFeature } from "../utils/feature.flag";
-import { ApplicationData } from "model";
+import { ApplicationData } from "../model";
```

Comment:
Does this need to change? Was something broken, with the introduction in this PR of the Redis refactorings?

Reply:
👍 I'd just add a comment to testers on the JIRA ticket to ensure Payment journey is regression tested.

Reply:
Yes, absolutely. Will mention this on the JIRA ticket so that the payment journey is also regression tested when the flag is OFF. We have a ticket (ROE-2732) about a payment issue when the flag is ON that I'll be working on immediately after this PR is approved.

Reply:
Thanks @mwejuli-ch .